protocol/driver/driver.go

```diff
@@ -148,12 +148,15 @@ func (c *parseNativeASTCommand) Execute(args []string) error {
 
 	nc, err := native.ExecClient(c.NativeBin)
 	if err != nil {
-		return fmt.Errorf("error executing native client: %s", c)
+		return fmt.Errorf("error executing native client: %s", err.Error())
 	}
 
 	defer func() { _ = nc.Close() }()
 
-	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{Content: string(b)})
+	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{
```

Comment:
I don't think that moving style changes to its own commit increases the workload of the reviewers, quite the opposite. Precisely by not separating them the author is delegating the burden of such hunt to the reviewers.